content/toolsAndResources/thePlayground.md

```diff
@@ -1,139 +1,125 @@
 keywords: babylon.js, tools, resources, playground
 further-reading:
-    - title: Textures Directly Available to the Playground
-      url: /toolsAndResources/assetLibraries/availableTextures
-    - title: Meshes Available to Import into the Playground
-      url: /toolsAndResources/assetLibraries/availableMeshes
-    - title: Using External Assets in the Playground
-      url: /toolsAndResources/thePlayground/externalPGAssets
-    - title: How to Use The Inspector in Projects
-      url: /toolsAndResources/inspector
-    - title: How to Use The Inspector in Projects
-      url: /toolsAndResources/inspector
-    - title: Make your own Snippet Server
-      url: /toolsAndResources/thePlayground/yourOwnSnippetServer
+  - title: Textures Directly Available to the Playground
+    url: /toolsAndResources/assetLibraries/availableTextures
+  - title: Meshes Available to Import into the Playground
+    url: /toolsAndResources/assetLibraries/availableMeshes
+  - title: Using External Assets in the Playground
+    url: /toolsAndResources/thePlayground/externalPGAssets
+  - title: How to Use The Inspector in Projects
+    url: /toolsAndResources/inspector
+  - title: How to Use The Inspector in Projects
+    url: /toolsAndResources/inspector
+  - title: Make your own Snippet Server
+    url: /toolsAndResources/thePlayground/yourOwnSnippetServer
 video-overview:
 video-content:
 ---
 
-
 ## The Playground
 
 The place to try out coding with Babylon.js. [Playground](https://playground.babylonjs.com/)
 
-Experimenting and changing any code in the playground and clicking on the **Run** button will not affect any original code in the playground you currently using. 
+Experimenting and changing any code in the playground and clicking on the **Run** button will not affect any original code in the playground you currently using.
 Original code can be restored by refreshing the browser.
 
 You can write the code in JavaScript or Typescript. The playground software compiles the code to JavaScript, in the background, before rendering.
 
+### Scene creation functions
+
+Usually you'll want to place your scene creation code in the already defined `createScene` function. But sometimes, you might want to substitute it for the `delayCreateScene` function, which allows you to return a scene without a camera (because for instance you plan to create the camera after an asynchronous mesh load, like: <Playground id="TVHK90#113" title="delayCreateScene example" description="An example of a Playground scene defined using the delayCreateScene function" image="/img/playgroundsAndNMEs/delayCreateSceneExample.png">).
+
+You can also override the engine creation by defining the `createEngine` function. This can be useful if you want to test a scene without antialiasing, for example: <Playground id="#NCWBUU#1" title="createEngine example" description="How to use createEngine on the playground to customize engine creation" image="/img/playgroundsAndNMEs/createEnginePG.png">

```

Comment:
```suggestion
You can also override the engine creation by defining the `createEngine` function. This can be useful if you want to test a scene without antialiasing, for example: <Playground id="#NCWBUU#1" title="createEngine example" description="How to use createEngine on the playground to customize engine creation" image="/img/playgroundsAndNMEs/createEnginePG.png"/>
```